UI/frontend-plugins/frontend-tools/auto-scene-switcher.cpp

```diff
@@ -128,9 +128,12 @@ SceneSwitcher::SceneSwitcher(QWidget *parent)
 		SetStopped();
 
 	loading = false;
+
+	QObject::connect(this, SIGNAL(finished(int)), this,
+			 SLOT(finished(int)));
```

Comment:
It would be nice to use new syntax (along the lines of `connect(this, &QDialog::finished, this, &SceneSwitcher::finished)`)